example_scripts/ANChor_plots_of_NSea_wvel.py

```diff
@@ -25,17 +25,23 @@
 dir_nam = '/projectsa/NEMO/jelt/AMM60_ARCHER_DUMP/AMM60smago/EXP_NSea/OUTPUT/'
 fil_nam = 'AMM60_1h_20120204_20120208_NorthSea.nc'
 
-chunks = {"x":10, "y":10, "time_counter":10}
-sci_w = coast.NEMO(dir_nam + fil_nam, 
-                 dom_nam, grid_ref='w-grid', multiple=True, chunks=chunks)
+config = "/work/jelt/GitHub/COAsT/example_files/example_w_nemo_config.json"
 
+chunks = {"x_dim":10, "y_dim":10, "t_dim":10}
```

Comment:
the chucking is now stored within the config file.

Reply:
It is. Though chunking is one of the things that it is nice to be able to play about with (and you can with xarray.chunk()), as you don't know how to chunk it until you've tried. I'll make a comment to this effect in the script.